assembly_knative-OCP-4x.adoc

```diff
@@ -0,0 +1,13 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+
+[id='knative-ocp-4x_{context}']
+= Knative on an OpenShift cluster 4.0
+
+// Knative on OpenShift cluster overview
+include::con_knative-OCP-4x.adoc[leveloffset=+1]
+
+// Installing Knative on a Minishift cluster using `install-on-minishift.sh`

```

Comment:
Again, Minishift shouldn't be part of OCP 3.11 or 4.0 assemblies, just in the master.adoc (its own topic)